Sources/mpc-core-kit-swift/mpcCoreKitSwift.swift

```diff
@@ -140,12 +142,43 @@ public struct MpcCoreKit  {
         return try await self.login(userData: TorusKeyData(torusKey: torusKey, userInfo: modUserInfo))
     }
     
+    public func getUserInfo() throws -> [String: Any] {
+        guard let userInfo = self.userInfo else {
+            throw ("user is not logged in.")
+        }
+        return userInfo
+    }
+    
+    public func getKeyDetails() async throws -> MpcKeyDetails {
+        guard let finalKeyDetails = try self.tkey?.get_key_details() else {
+            throw ("Key Details Not Found!")
+        }
+        if((self.tkey == nil)) {
+            throw ("Tkey is not initialized!")
+        }
+        let tssTag = try TssModule.get_tss_tag(threshold_key: self.tkey!)
+        let tssPubKey = try await TssModule.get_tss_pub_key(threshold_key: self.tkey!, tss_tag: tssTag)
```

Comment:
You'll also need the tss index. 

Consider returning this as a sub struct of:
```
tss_tag
tss_index
verifier
verifier_id
```

However what you return here will depend on what you want to use it for, is it just for display?

Reply:
Since multiple tss-tag can exist, this struct will also no be singular but a sub item in an array.

Reply:
following the same return type as web version let me know @ieow we can implement this too in web first?

Reply:
I think we would not encourage client to use the tags as we now have soft-nonce index for the multi-wallet.